### PR TITLE
Update anypoint-exchange-release-notes.adoc

### PR DESCRIPTION
**Pull request by joaquinstan**
Update release notes to show new discard draft feature for April 2019

### CODE REVIEW

**Comment by wwarner-salesforce** [comment]
Looks good! @joaquinstan is this feature live in production?

**Comment by joaquinstan** [comment]
Not yet @wwarner-salesforce, will let you know when it is, thanks for the fixes!

**Comment by wwarner-salesforce** [comment]
You're welcome, and thank you for letting me know about the timing!

**Comment by joaquinstan** [comment]
@kuntald let me know what you think!

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc [hunk 1]
```diff
@@ -3,6 +3,12 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
+== April 2019
+
+=== New Features
+
+* Enable customers to discard drafts of portal documentation. (EXC-3116)
```
Comment (kuntald) [inline modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: * Users now have a way to delete a draft of the portal documentation. Prior to this release, the only way a user could delete a draft was to edit it, then remove all the new edits & publish the documentation. But if there were multiple changes in the draft, it was very difficult to ascertain which edits belonged to the draft, vs which edits belonged to the old version. Hence giving the option to the users to delete a draft, gives users the confidence that the right changes will be automatically discarded without any manual intervention, preserving the original documentation content.

Comment (wwarner-salesforce) [reply modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Much better! I updated the text based on this info. Thank you, Kuntal!

**Comment by wwarner-salesforce** [comment]
If no one objects I'll merge this later today.

**Comment by wwarner-salesforce** [comment]
My mistake! Joaquin explained that the release is tomorrow, so I'll merge then.

**Comment by wwarner-salesforce** [comment]
Will publish Monday per update from the team in today's meeting.

**Comment by wwarner-salesforce** [comment]
That looks great, Kuntal. Thank you very much!

**Comment by wwarner-salesforce** [comment]
If no one has any further changes, I will merge and publish this version.

**Pull request merged**